src/pages/components/menu/style.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Menu
+description:
+  A menu is a list of interactive options that appears when users interact with
+  an element or perform a specific action.
+tabs: ['Usage', 'Style', 'Code', 'Accessibility']
+---
+
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink> <AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink> <AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Color
+
+### Default state
```

Comment:
I believe we do not need the “Default state” heading, and only need the H2 "Color" as the heading here. (according to Aubrey's style tab content template)